src/templates/basic.ts

```diff
@@ -3,7 +3,6 @@ import { Template } from './template';
 const basicTemplate: Template = {
   name: 'basic',
   dependencies: [
-    'husky',
```

Comment:
what do you think keep the `husky` here but removing it when `opts.husky` is `false`? I'm asking it as it's the default.

Reply:
That makes more sense given how it's an opt-out rather than an opt-in. Will work on that.

Reply:
Done.